### PR TITLE
Replace long timeout with Duration for ElementsCollection 

### PR DESCRIPTION
**Pull request by ostap-oleksyn**
## Proposed changes
Since recently all 'should' methods in SelenideElement have a Duration argument for condition waiting. To make things consistent these changes update all 'should' methods for ElementsCollection to also use Duration instead of long for timeouts.

## Checklist
- [x] Checkstyle and unit tests are passed locally with my changes by running `gradlew check chrome_headless firefox_headless` command
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by asolntsev** [comment]
@ostap-oleksyn Thank you. It's a good idea to use `Duration`, but shouldn't we also keep the old methods with `long` argument for backward compatibility? At least for some time. We could mark them as deprecated.

**Comment by ostap-oleksyn** [comment]
> @ostap-oleksyn Thank you.
> 
> It's a good idea to use `Duration`, but shouldn't we also keep the old methods with `long` argument for backward compatibility? At least for some time. We could mark them as deprecated.
> 
> P.S. Don't pay attention to the failing tests: they were also failing in master. Already fixed in master by now.

Good point. I'll update it.

File: statics/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0119/221532.796:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
```
Comment (rosolko) [inline statics/debug.log]: Please do not commit debug file

Comment (ostap-oleksyn) [reply statics/debug.log]: already removed it

**Review by asolntsev — APPROVED** [review]

**Pull request merged**